workspaces/liferay-marketplace-workspace/client-extensions/dashboard-remote-app/src/Dashboard.css

```diff
@@ -0,0 +1,12 @@
+.table .column-name:hover {
+    cursor: pointer;
+}
+
+.table .column-name:hover:after {
+    content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' class='lexicon-icon lexicon-icon-order-arrow' height='12' width='12' role='presentation' viewBox='0 0 512 512'%3E%3Cpath class='lexicon-icon-outline order-arrow-arrow-down' d='M307.6,431.5l71.6,71.6c11.9,10.8,32.9,11.2,45.8,0l71.6-71.6c29.8-33.6-15.8-74.9-45.8-45.8l-16.7,16.7V31.6c0-42.5-64-42.5-64,0v370.8l-16.7-16.7C321.6,355.8,277.9,400.6,307.6,431.5z' fill='%23999AA3'%3E%3C/path%3E%3Cpath class='lexicon-icon-outline order-arrow-arrow-up' d='M204.4,80.5L132.8,8.9C120.9-1.8,99.9-2.3,87,8.9L15.4,80.5c-29.8,33.6,15.7,74.9,45.8,45.8l16.7-16.7v370.8c0,42.5,64,42.5,64,0V109.6l16.7,16.7C190.4,156.2,234.1,111.4,204.4,80.5z' fill='%23999AA3'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
I find these always hard to read - is there a way we could just use a regular svg in markup instead of this?